MediaEditor.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MediaEditor'
-  s.version          = '1.2.0'
+  s.version          = '1.2.1-beta.1'
```

Comment:
I don't think anything else will be added in `1.2.1`, we can release it right away with this fix (without using a beta Pod). Wdyt?

Reply:
Sure!